actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
I agree. We see that the "moved" messages can become performance impacting when a lot of nodes have outdated caches so we should try and have them evict from the cache around the same time as the actor is likely to be deactivated.